addons/binding/org.openhab.binding.hs110/README.md

```diff
@@ -0,0 +1,41 @@
+# openhab2-addon-hs110
+OpenHAB Addon for TP-Link HS100/HS110
+
+
+# Issues and Contribution
+Development happens on the following github project:
+https://github.com/computerlyrik/openhab2-addon-hs110
```

Comment:
I think that is not the way it should normally work when you contribute but others might have opinion about this @kaikreuzer ?